plotting/timeseries.py

```diff
@@ -177,13 +192,15 @@ def csv(self):
             else:
                 header_text = "%s (%s) %s (%s) %s (%s) %s" % (
                     self.variable_name, self.variable_unit,
-                    self.variable_names[0], self.variable_units[0],
-                    self.variable_names[1], self.variable_units[1],
+                    self.vector_variable_names[0], self.vector_variable_units[0],
+                    self.vector_variable_names[1], self.vector_variable_units[1],
                     "Bearing (degrees clockwise positive from North)"
                 )
                 header.append(["Variables", header_text])
-                for var_name in [self.variable_name, self.variable_names[0],
-                                 self.variable_names[1], "Bearing"]:
+                for var_name in [self.variable_name, 
+                                 self.vector_variable_names[0], 
+                                 self.vector_variable_names[1],
+                                 "Bearing"]:
```

Comment:
Cool trick: you can use `*my_list` to unpack a list into another list :exploding_head: and write less code
```suggestion
                for var_name in [self.variable_name, *self.vector_variable_names, "Bearing"]:
```

Go to https://www.programiz.com/python-programming/online-compiler/
and paste and run to play with the syntax:
```python
test = [1, 2, 3]

more = [4, 5, *test]

print(more)

```